raindrops.json

```diff
@@ -0,0 +1,106 @@
+{
+   "#": [
+     "Language implementations vary when one attempts a prime factor on a number with",
+     "and underscore in it.",
+     "In this set of test cases, we have one of these, and the solution would need to",
+     "handle the language implementation."
+   ],
+   "cases": [
+      {
+         "description": "1",
+         "number" : "1", 
+         "expected": "1",
+         "msg": "No, raindrop-speak of 1 is 1."
+      },
+      {
+         "description": "3",
+         "number" : "3", 
+         "expected": "Pling",
+         "msg": "No, raindrop-speak of 3 is Pling."
```

Comment:
These messages don't really add any value, they're just an echo of the implementation. Let's ditch it.